py3-aiosignal.yaml

```diff
@@ -7,18 +7,25 @@ package:
   copyright:
     - license: Apache-2.0
   dependencies:
-    runtime:
-      - py3-frozenlist
-      - python3
+    provider-priority: 0
```

Comment:
need to bump epoch

Reply:
Yeah I should've put this in draft mode a while ago since it can't find happy eyes for the build.